### DISCUSSION TITLE
HELP ME PLS: Incorrect Content-Type: must be textual to extract_string, JSON to extract_json.

### DISCUSSION THREAD

**Discussion by BleakFox** [powenn/AltServer-Linux-ShellScript#60] [Q&A]
Hello, when I click "i" then "1" (install alt store) and then I select to use a saved Apple ID account, the code in output is that:

```
Received response status code: 500
parse anisette data ret
Alert: Could not install AltStore.ipa to unknown.
   Incorrect Content-Type: must be textual to extract_string, JSON to extract_json.
Press any key to continue...

Exception: Incorrect Content-Type: must be textual to extract_string, JSON to extract_json.
0# 0x00000000005CF298
1# 0x000000000099EC2C
```

The same thing happens when I try to DON'T use a saved Apple ID account, but the error code is the same... please can anyone help me? I'm already using the last version of the code in ubuntu 22.04 terminal, and the last Alt Store release in IOS..

**Comment by powenn** [comment]
https://github.com/powenn/AltServer-Linux-PyScript

Use this instead
